lldb/source/Symbol/SymbolContext.cpp

```diff
@@ -163,7 +166,10 @@ bool SymbolContext::DumpStopContext(Stream *s, ExecutionContextScope *exe_scope,
       dumped_something = true;
       if (symbol->GetType() == eSymbolTypeTrampoline)
         s->PutCString("symbol stub for: ");
-      symbol->GetName().Dump(s);
+      if (pattern)
+        Address::DumpName(s, symbol->GetName().GetStringRef(), pattern);
+      else
+        symbol->GetName().Dump(s);
```

Comment:
Same here, I think you can just always call Address::DumpName and it'll take the fast path if pattern is nullptr.

Reply:
✅

Reply:
This one still needs fixing, can be one call to PutCString... I think.

Reply:
When I opened this suggestion, it only showed the changes from the first commit, not "changes from all commits", maybe that's the reason it was not corrected yet. I don't know why that happened.

Reply:
This suggestion got lost along the way and still needs doing, but you'll call `PutCStringColorHighlighted` now.